database/settings.json

```diff
@@ -1 +1 @@
019 will now begin!","application_success_email_nl":"Je hebt je zojuist aangemeld via onze website voor de twaalfde editie van Het Weekend van de Rollende Keukens. De selectieprocedure gaat nu van start. \r\n\r\nLet op! Dit is geen bevestiging van deelname.\r\n\r\nHet is mogelijk om je ingevulde gegevens terug te zien, en\/of aan te passen met je mailadres en wachtwoord.","application_success_email_en":"We have received your application for the twelfth edition of The Weekend of The Rolling Kitchens. The selection procedure will now begin.\r\nTake notice! This is not a confirmation of participation.\r\nIt is possible to view and\/or modify your data by logging in with your mail and password.","invoices_accountant":"jstrosin@yahoo.com","invoices_business_details":"Voluptatem molestiae consequatur est dolor non deleniti. Repudiandae omnis quisquam omnis totam facilis rerum. Quo qui ad veniam quod. Et similique qui corporis quasi.","invoices_default_subject_nl":"Perferendis rerum maxime minima optio mollitia. Nisi cum numquam dicta reprehenderit et iste consequuntur. Voluptatem voluptatem et labore dolorum illum fugit.","invoices_default_subject_en":"Et voluptas modi earum est similique officia. Dolores itaque consequatur sint voluptas.","invoices_default_email_nl":"Facere distinctio odio voluptas aut ea. Eum quod voluptas et aut dolorem est officia. Voluptas illo quaerat qui velit.","invoices_default_email_en":"Ut molestiae vitae eos consectetur. Esse laudantium veniam rem beatae.","invoices_default_resend_subject_nl":"Dolores quaerat ut ut quam. Illum nam aspernatur nihil atque ullam nostrum. Reprehenderit impedit ullam ut sed.","invoices_default_resend_subject_en":"Voluptatem aperiam aliquam minima voluptas vitae. Voluptas eaque delectus quisquam dolorem ea doloremque. Tenetur et ex consequatur quae. Et pariatur ut qui et id delectus quo.","invoices_default_resend_email_nl":"Nihil voluptas consequatur tempore mollitia. Itaque doloremque aut facere molestias repudiandae sit in quas. Soluta architecto quia illo porro eum.","invoices_default_resend_email_en":"Maxime pariatur ut voluptas illo. Eos aliquam voluptatibus delectus quam pariatur ut odio. Aliquam et voluptates animi. Ut non sed nostrum aut possimus. Dolorem nam ea animi vel labore aut.","invoices_notes_nl":"Quasi illum ullam beatae deleniti et. Neque sit vel corporis omnis dolor. Eligendi ullam aut animi modi unde nihil ut. Eum quia ut laborum. Autem eligendi voluptatibus blanditiis sint libero commodi.","invoices_notes_en":"Earum perspiciatis voluptatibus laborum. Aut enim vitae voluptas ipsum id dolores. Rerum unde ut nemo. Omnis vitae totam consectetur.","invoices_footer_nl":"Rerum illo explicabo quas asperiores necessitatibus voluptas ea. Quae ullam quia autem aut. Porro laudantium aut architecto blanditiis.","invoices_footer_en":"Necessitatibus ab repudiandae laudantium sit. Voluptas hic molestiae totam molestiae numquam eaque. Aut ut quia doloremque ea sint sequi reiciendis impedit."}
```

Comment:
please don't commit the settings -it messes with theirs. we should actually completely stop following those